crates/zk-por-core/src/circuits/merkle_sum_circuit.rs

```diff
@@ -0,0 +1,124 @@
+use plonky2::{
+    hash::{
+        hash_types::HashOutTarget,
+        poseidon::PoseidonHash,
+    },
+    iop::target::Target,
+    plonk::circuit_builder::CircuitBuilder,
+};
+
+use crate::types::{F,D};
+
+use super::{account_circuit::AccountSumTargets, circuit_utils::assert_non_negative_unsigned};
+
+/// A node in the merkle sum tree, contains the total amount of equity (in usd) and the total amount of debt (in usd) and the hash.
+/// 
+/// The hash is Hash(hash_left, hash_right).
+/// 
+/// The amount of equity and amount of debt is the sum of the equity and debt of the children.
+#[derive(Debug, Copy, Clone,)]
+pub struct MerkleSumNodeTarget{
+    pub sum_assets: Target,
+    pub sum_debt: Target,
+    pub hash: HashOutTarget
+}
+
+impl MerkleSumNodeTarget {
+    /// Given two children, generate the next MerkleSumNode
+    pub fn get_child_from_parents(
```

Comment:
the naming is not consistent with comment

Reply:
Fixed